drivers/media/i2c/irs1125.h

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
I'm not sure about this file. I use this one also from user space (currently I just copy it into my user space project), but as far as I remember there is also a location, maybe include/video to put these header files which are shared between kernel and user space?

Reply:
/include/uapi/linux
Once "published" then uapi defines are pretty much set in stone to avoid regressions, so ensure you've got it right before you push it to there.

That's the correct way, but you'll often find that applications end up having to include the latest and greatest headers from the point at which they were built in order to avoid a #ifdef hell trying to remove code that relies on it.

Reply:
Okay, than I definitely will leave it in drivers/media/i2c
We are only using a subset of the imager's possibilities so far.